modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/provider/BaseSegmentsEntryProvider.java

```diff
@@ -359,12 +367,8 @@ private Map<String, Object> _getUserAttributes(User user) throws Exception {
 		).put(
 			"groupIds", user.getGroupIds()
 		).put(
-			"inheritedRoleIds",
-			TransformUtil.transform(
-				user.getInheritedRoles(), role -> role.getRoleId()
-			).toArray(
-				new Long[0]
-			)
+			"inheritedUserGroupRoleIds",
```

Comment:
@rachaelkoestartyo, I think we should also consider those roles that are assigned to Organizations, Sites and Segments as inherited roles.